README.rst

```diff
@@ -79,7 +79,10 @@ and then build build the extension modules and run the  test suite:
 .. code-block:: shell-session
 
     python setup.py develop
-    pytest tests/
+    pytest tests/ --import-mode=prepend
```

Comment:
What happens if we don't set this in `setup.cfg`? Can we avoid having to type this explicitly when running locally and explicitly set `--import-mode` on Travis CI?